components/server/src/ome/services/ThumbnailBean.java

```diff
@@ -865,8 +865,7 @@ private Thumbnail _createThumbnail(Thumbnail thumbMetaData) {
         try {
             if (thumbMetaData == null) {
                 throw new ValidationException("Missing thumbnail metadata.");
-            } else if (ctx.dirtyMetadata(pixels.getId()) &&
-                    thumbMetaData.getDetails().getOwner() != null) {
```

Comment:
@jburel and @will-moore point out that this may in fact need to be on the old method as well.